spec/configuration/sign_in_token_option_spec.rb

```diff
@@ -21,7 +21,8 @@
         controller_class.acts_as_token_authentication_handler_for User
 
         @controller = controller_class.new
-        allow(@controller).to receive(:params)
+        allow(@controller).to receive(:params).and_return({})
+        allow(@controller).to receive_message_chain(:request, :headers).and_return({})
```

Comment:
**Note to self** (not necessarily related to the PR): Check this, there may be a missing example here.